### ISSUE TITLE
loss when end training

### ISSUE THREAD

**Issue by pageedward** [georgesung/ssd_tensorflow_traffic_sign_detection#43]
what is your loss when ended training?

**Comment by CNHNLP** [comment]
You can increase the learning rate.I get the loss 500+ by using the default learning rate 0.01.But the result is better if you set the learning rate 0.5
